apps/remix-ide/src/blockchain/blockchain.tsx

```diff
@@ -1,6 +1,7 @@
 
 import React from 'react' // eslint-disable-line
 import Web3 from 'web3'
+import { fromWei, toBN, toWei } from 'web3-utils-legacy'
```

Comment:
This was a temporary decision made to reduce the friction of migrating to `4.x` since `BN` is no longer used

Reply:
I've removed this dependency